src/briefcase/bootstraps/toga.py

```diff
@@ -112,6 +117,9 @@ def pyproject_table_linux_system_suse(self):
     "cairo-devel",
     # Needed to compile PyGObject wheel
     "gobject-introspection-devel",
+    # Needed to run the app
+    "gtk3", "typelib-1_0-Gtk-3_0",
+    # "libwebkit2gtk3", "typelib-1_0-WebKit2-4_1",
```

Comment:
Minor nit: these should be broken out by line, and explain why webkit is commented out:
```suggestion
    "gtk3", 
    "typelib-1_0-Gtk-3_0",
    # Needed if your app uses WebView
    # "libwebkit2gtk3", 
    # "typelib-1_0-WebKit2-4_1",
```

Reply:
n/a. Updated Dockerfile handling to use both `system_requires` and `system_runtime_requires` when it creates the image.